make/go-header.txt

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
?

Reply:
same as in https://github.com/codeready-toolchain/api/blob/master/make/go-header.txt#L2 but let me remove that line and just keep the License part.